addons/website_blog/tests/test_ui.py

```diff
@@ -9,4 +9,11 @@
 class TestUi(odoo.tests.HttpCase):
     def test_admin(self):
         self.env['blog.blog'].create({'name': 'Travel'})
+        self.env['ir.attachment'].create({
+            'public': True,
+            'type': 'url',
+            'url': '/web/image/123/transparent.png',
```

Comment:
can we have a real picture, to avoid potential 404 error and more in log ?

Reply:
does this not simply return the placeholder image when there is no attachment corresponding to the id? I don't get any 404 in the logs with this, regardless of the id. If you still want me to use a real image, do you have a suggestion that isn't a /static/ image? Since it needs to show up in the media-dialog